package.json

```diff
@@ -16,52 +16,61 @@
   },
   "repository": "https://github.com/adopted-ember-addons/ember-page-title",
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
     "release": "standard-version"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.7.3",
+    "ember-cli-babel": "^7.21.0",
     "ember-cli-head": "^1.0.0",
-    "ember-cli-htmlbars": "^3.0.1",
+    "ember-cli-htmlbars": "^5.2.0",
```

Comment:
Do we actually have any templates in addon/? If not, can you move this to devDeps?

Reply:
It's here because of ember-cli-head -> head.hbs in app/templates folder.

Reply:
Gotcha. So the other PR will remove that template and move this to devDeps?

Reply:
If we get this sorted out: https://github.com/adopted-ember-addons/ember-page-title/pull/168 - then yes.